DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Utility.cs

```diff
@@ -0,0 +1,181 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.DeID.SharedLib.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core
+{
+    /// <summary>
+    /// Utility functions are used to parse DICOM data including DA, DT, AS. The format for these VR is given in DICOM standard.
+    /// http://dicom.nema.org/medical/Dicom/2017e/output/chtml/part05/sect_6.2.html
+    /// </summary>
+    public static class Utility
```

Comment:
maybe we can have some [extension classes](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods) like AgeExtension, DicomDateExtension, DimcomDateTimeExtension instead of a huge utility class.

Reply:
We can refine it next PR.